Added different sort options, and fixed infinite runtime bug

makeworld-the-better-one opened the pull request:
I've added second and third arguments so that you can choose the sort of the page. The arguments `aww top all` will return the top posts of all time from r/aww, and `aww new` will start searching for images on the new page.

I also noticed that the script runs indefinitely, because the `if [ -z $after ]` check doesn't work, there will always be a value for `$after`. The correct bad value is `null`, so I added a check for that and it successfully exits now.

ostrolucky commented on download-subreddit-images.sh:
```diff
@@ -24,10 +35,12 @@ while : ; do
+    if [[ -z $after ]]; then
+        break
+    fi
+    if [[ $after == "null" ]]; then
```
Thx for report, I fixed the problem in bf7ef7db8c6670edda827a7c96a6ce8c4e77dac9. You may remove this condition now.

makeworld-the-better-one commented:
I've updated the README, and removed the extra condition. This PR now only adds functionality for sorting by page, and having different time scales for the top sort.

ostrolucky commented:
merged in 0046879, thanks
